build.rs

```diff
@@ -20,15 +20,29 @@ fn link_cuda() {
     #[cfg(feature = "driver")]
     println!("cargo:rustc-link-lib=dylib=cuda");
     #[cfg(feature = "driver")]
-    println!("cargo:rustc-link-lib=dylib=cudart");
+    println!("cargo:rustc-link-lib=static=cudart_static");
     #[cfg(feature = "nvrtc")]
     println!("cargo:rustc-link-lib=dylib=nvrtc");
     #[cfg(feature = "curand")]
     println!("cargo:rustc-link-lib=dylib=curand");
-    #[cfg(feature = "cublas")]
-    println!("cargo:rustc-link-lib=dylib=cublas");
-    #[cfg(feature = "cublas")]
-    println!("cargo:rustc-link-lib=dylib=cublasLt");
+
+    println!("cargo:rerun-if-env-changed=STATIC");
+    let is_static = std::env::var("STATIC").unwrap_or_else(|_| "0".to_string()) == "1";
```

Comment:
Let's go with feature flag, it's more reproducible than environment variable and I think fits better within rust ecosystem at least.